modules/core/src/main/java/org/locationtech/jts/io/CheckOrdinatesFilter.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2016 Vivid Solutions.
```

Comment:
this is incorrect for a new file

Reply:
All source code in this file is taken from `WKTWriter.java`, which is an old file. Should we update the copyright header or preserve the copyright header of the file where it originates from?